examples/electron/README.md

```diff
@@ -27,7 +27,7 @@ let mainWindow;
 
 function createWindow() {
     let container = desktopJS.resolveContainer();
-    mainWindow = container.createWindow('http://localhost:8000');
+    container.createWindow('http://localhost:8000').then(win => mainWindow = win);
```

Comment:
Ideally this should be async/await for easier reading and code but hindered by current browser usage.  This can also be mitigated by changing the examples to ts and targeting es5 but that is significant effort for little reward.